src/BaGet.Core/Search/VersionsRequest.cs

```diff
@@ -0,0 +1,29 @@
+namespace BaGet.Core
+{
+    /// <summary>
+    /// The NuGet V3 enumerage package versions request.
+    /// See: https://docs.microsoft.com/en-us/nuget/api/search-autocomplete-service-resource#request-parameters-1
+    /// </summary>
+    public class VersionsRequest
+    {
+        /// <summary>
+        /// Whether to include pre-release packages.
+        /// </summary>
+        public bool IncludePrerelease { get; set; }
+
+        /// <summary>
+        /// Whether to include SemVer 2.0.0 compatible packages.
+        /// </summary>
+        public bool IncludeSemVer2 { get; set; }
+
+        /// <summary>
+        /// Filter results to a package type. If null, no filter is applied.
+        /// </summary>
+        public string PackageType { get; set; }
```

Comment:
Please remove the `PackageType` property as the autocomplete package versions API doesn't accept a package type filter (see [this](https://docs.microsoft.com/en-us/nuget/api/search-autocomplete-service-resource#request-parameters-1)).